cypress/e2e/shared/explorer.test.ts

```diff
@@ -1526,4 +1526,90 @@ describe('DataExplorer', () => {
       // TODO: add filter values based on dropdown selection in key / value
     })
   })
+
+  describe('simple table interactions', () => {
+    const simpleSmall = 'simple-small'
+    const simpleLarge = 'simple-large'
+    beforeEach(() => {
+      cy.window().then(win => {
+        win.influx.set('simpleTable', true)
+      })
+
+      cy.get('@org').then(({id: orgID}: Organization) => {
+        cy.getByTestID('tree-nav')
+        cy.createBucket(orgID, name, simpleLarge)
+        cy.writeData(lines(300), simpleLarge)
+        cy.createBucket(orgID, name, simpleSmall)
+        cy.writeData(lines(30), simpleSmall)
+        cy.reload()
+      })
+    })
+
+    it('should render correctly after switching from a dataset with more pages to one with fewer', () => {
+      cy.getByTestID('query-builder').should('exist')
+
+      // show raw data view of data with 100 pages
+      cy.getByTestID(`selector-list ${simpleLarge}`).should('be.visible')
+      cy.getByTestID(`selector-list ${simpleLarge}`).click()
+
+      cy.getByTestID('selector-list m').should('be.visible')
+      cy.getByTestID('selector-list m').clickAttached()
+
+      cy.getByTestID('selector-list v').should('be.visible')
+      cy.getByTestID('selector-list v').clickAttached()
+
+      cy.getByTestID('selector-list tv1').clickAttached()
+
+      cy.getByTestID('time-machine-submit-button').click()
+
+      cy.getByTestID('raw-data--toggle').click()
+      cy.getByTestID('table').should('exist')
```

Comment:
Table isn’t quite clear on which table we’re testing. I know the test suite is designed for simple table, but someone might have set the testID for the other tables as just “table”. Might be worth being a little more specific just in case?

Reply:
changed to `simple-table`